src/amo/components/AddonReviewListItem/index.js

```diff
@@ -304,9 +304,11 @@ export function mapStateToProps(
   if (ownProps.review) {
     const view = state.reviews.view[ownProps.review.id];
     if (view) {
+      /* eslint-disable prefer-destructuring */
```

Comment:
I would rather not have to use this comment for assignments everywhere. I think you can turn this off with [`{AssignmentExpression: {array: false, object: false}}`](https://eslint.org/docs/rules/prefer-destructuring). Did you try that?

Reply:
Cool, I'll give that a spin.